guides/v2.1/install-gde/prereq/mysql.md

```diff
@@ -55,11 +55,15 @@ To install MySQL 5.7 on Ubuntu 16:
 
 		sudo apt install -y mysql-server mysql-client
 
-2.	Secure the installation.
+2. Start MySQL:
+
+        sudo service mysql start
+
+3.	Secure the installation.
```

Comment:
These line items also need colons...

`Secure the installation:`

I will add them. But lists, ordered or unordered, need to be consistent.